src/ontology/mondo-ingest.Makefile

```diff
@@ -187,6 +190,16 @@ config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/o
 	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
 	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
 
+# todo: Is it desired for there to be merged `.ttl` and `.obo` files? How to do that? Pass multiple `--input` and use the merged `config/exclusion_reasons.robot.template.tsv` for --template?
+# TODO: add back these prerequisites to this target:
+# config/%_exclusion_reasons.ttl: mirror/%.owl config/%_exclusion_reasons.robot.template.tsv
+config/%_exclusion_reasons.ttl:
+	robot template --input mirror/$*.owl --template config/$*_exclusion_reasons.robot.template.tsv --output config/$*_exclusion_reasons.ttl
+
+# TODO: Fix error: Optional.get() cannot be called on an absent value
+config/%_exclusion_reasons.obo: mirror/%.owl config/%_exclusion_reasons.robot.template.tsv
```

Comment:
@matentzn 
I ran `sh run.sh make config/doid_exclusion_reasons.obo`, but got this error:

> `Optional.get() cannot be called on an absent value`

I ran with `-vvv`. Here's the last, relevant part of the very long stack trace:
[optional get err.txt](https://github.com/monarch-initiative/mondo-ingest/files/9294640/optional.get.err.txt)

I ran for a couple ontologies and got the same issue, so I assume this will happen for any ontologies I try to run on this command, but not sure.

Reply:
Not doing OBO for now.